examples/deferred_processing/README.md

```diff
@@ -0,0 +1,340 @@
+    96439afdaf8249be81773c89fc87b366 edges_dp           31 seconds ago     3 seconds 0       1 + 4 / 5 204KiB   0B       success 
+    4b05f2c0ddf14ab0a4ca8f4ee2d6d9ff montage_dp         About a minute ago 3 seconds 0       1 + 0 / 1 693.8KiB 673.6KiB success 
+    1a8d2fefc2ba450e9102ffd6b74e503d montage_dp         3 minutes ago      4 seconds 0       1 + 0 / 1 693.8KiB 673.6KiB success 
+    c5feebe5c6ae48e0adc4f5bd7be9a9ac edges_dp           3 minutes ago      3 seconds 0       1 + 3 / 4 175.1KiB 0B       success 
+    7d6aa9bbe1484e2e9ac60a7f84147e1b montage_dp         24 minutes ago     1 second  0       0 + 0 / 0 0B       0B       success 
+    f7a7d71dd7e74a3f9c48d4b9d2897ba3 edges_dp           24 minutes ago     4 seconds 0       2 + 1 / 3 181.1KiB 0B       success 
+    f9c620d1a4cc421a9c3923b749e9e6ed edges_dp           24 minutes ago     3 seconds 0       1 + 0 / 1 57.27KiB 0B       success 
+    ```
+
+13. We can create another "stable" branch called `stable_1_1`.
+    
+    ```sh
+    $ pachctl create branch edges_dp@stable_1_1 --head master
+    ```
+
+14. We can move master to our old stable branch, `stable_1_0`,
+    which will create a job in `montage_dp` which skips its datum,
+    as that data was already processed.
+    
+    ```sh
+    $ pachctl create branch edges_dp@master --head stable_1_0
+    $ pachctl list job
+    ID                               PIPELINE           STARTED            DURATION  RESTART PROGRESS  DL       UL       STATE   
+    01bb8e400c054a18a5c2c691f0b5a65d montage_dp         11 seconds ago     2 seconds 0       0 + 1 / 1 0B       0B       success 
+    96439afdaf8249be81773c89fc87b366 edges_dp           42 seconds ago     3 seconds 0       1 + 4 / 5 204KiB   0B       success 
+    4b05f2c0ddf14ab0a4ca8f4ee2d6d9ff montage_dp         About a minute ago 3 seconds 0       1 + 0 / 1 693.8KiB 673.6KiB success 
+    1a8d2fefc2ba450e9102ffd6b74e503d montage_dp         3 minutes ago      4 seconds 0       1 + 0 / 1 693.8KiB 673.6KiB success 
+    c5feebe5c6ae48e0adc4f5bd7be9a9ac edges_dp           3 minutes ago      3 seconds 0       1 + 3 / 4 175.1KiB 0B       success 
+    7d6aa9bbe1484e2e9ac60a7f84147e1b montage_dp         24 minutes ago     1 second  0       0 + 0 / 0 0B       0B       success 
+    f7a7d71dd7e74a3f9c48d4b9d2897ba3 edges_dp           24 minutes ago     4 seconds 0       2 + 1 / 3 181.1KiB 0B       success 
+    f9c620d1a4cc421a9c3923b749e9e6ed edges_dp           24 minutes ago     3 seconds 0       1 + 0 / 1 57.27KiB 0B       success 
+    ```
+
+### Transactions
+
+15. If you want to run a particular set of data in `images_dp_2` 
+    against a particular branch of `edges_dp`,
+    you need to perform two operations
+    - commit data to `images_dp_2` and
+    - point `edges_dp@master` to the branch of interest.
+    
+    This will result in two jobs being triggered.
```

Comment:
```suggestion
    If you do not use a transaction, this will result in two jobs being triggered, one for the new commit and a second when we move `edges_dp@master` branch.
```